### DISCUSSION TITLE
Automated synchronization of model

### DISCUSSION THREAD

**Discussion by rosafaria** [iTwin/itwinjs-core#2678] [Q&A]
Hi all,

I need to update an iModel with a kml file I previously placed in a folder

I'm trying to build a connector from the sample in https://github.com/imodeljs/itwin-connector-sample (ignore the extractor)

I had to change the start script, since the class imodeljs-backend/lib/iModelBridgeFwkMain.js was not found. Instead I used the Main class in the imodel-bridge module.

This presented me with a problem: the authentication. I know an access token has to be passed (along with the modelId, etc), but, even after reading the documentation related with AccessToken, I didn't understand how to get it, since there was a reference to an Open ID Connect, but I couldn't find a module for it, or an api.
I try to run it but it doesn't even show errors.

What am I missing?

Thanks

**Comment by jffmarker** [comment]
Can you help @jchick-bentley?

**Comment by jchick-bentley** [comment]
@rosafaria This was just brought to my attention.  There's a new COBie connector-sample available at https://github.com/iTwin/connector-samples  The authentication should be encapsulated for you.  Please have a look at the new sample and documentation.  If you have any additional questions please post to https://github.com/iTwin/connector-samples/issues.  I am monitoring the discussions there.
